projects/addon-commerce/components/input-card/input-card.template.html

```diff
@@ -18,4 +16,10 @@
     (focusedChange)="onFocused($event)"
 >
     <ng-content></ng-content>
+    <input
```

Comment:
Let's instead allow `tuiTextfield` inside `InputCard` and make `InputCard` directive that would set initial `placeholder` if it's missing and set `inputmode`.

Reply:
done